### DISCUSSION TITLE
The Binding of Isaac: Repentance

### DISCUSSION THREAD

**Discussion by Motaart** [skyline-emu/skyline#516] [Game Requests]
THE BINDING OF ISAAC: REPENTANCE

**Comment by PixelyIon** [comment]
We technically already support this if you manually merge the DLC which is very complex and I'm not going to get into it, once we add DLC support to Skyline it should just work.

**Reply by Motaart** [reply]
How can I support the project? Do you have a donation link or something like that ?

**Reply by PixelyIon** [reply]
We appreciate it but at the moment we don't as we didn't want to accept sponsors till we got something substantial working. We likely will in the future though, I recommend sticking around in our [Discord Server](https://discord.gg/XnbXNQM) to stay updated.
